lib/sparql_to_sw_solr/instance_topic_fields.rb

```diff
@@ -0,0 +1,38 @@
+module SparqlToSwSolr
+  class InstanceSolrDoc
+    module InstanceTopicFields
```

Comment:
How about `TopicFields`/`topic_fields` because the actual field symbols begin with `:topic_*` and the SW-UI displays them in a `Topics` facet?